### PR TITLE
Adds in SCP-3349 (cardiac arrhythmia with a twist)

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Adds in [SCP-3349](https://scp-wiki.wikidot.com/scp-3349). Basically, by injecting the right chemicals, you can cause someone to permanently gain this condition where, sometimes, their pulse changes to feel/sound like a cat. If you put someone with this arrhythmia in a scanner while that's happening, you can print out a neat EKG report that science can deconstruct for shittons of tech.

Credit goes to the original author (who seems to have his account deleted) for the image and sound.

## Why It's Good For The Game

The more skips the better.

## Changelog

:cl:
add: Adds in SCP-3349, a keter-class cardiac arrhythmia! Get the science team on it for loadsa tech!
soundadd: SCP-3349 heartbeat
imageadd: SCP-3349 EKG report
code: carbon Life() has a signal now, please use it when possible.
/:cl:

### CODE REVIEW

**Comment by 77seven77seven** [comment]
so far looks pretty based

File: code/_helpers/medical_scans.dm [hunk 1]
```diff
@@ -165,12 +167,14 @@
 		<tr><td colspan="2" align="center"><span class='bad'>Patient in Hypovolemic Shock. Transfusion highly recommended.</span></td></tr>
 	*/
 	dat += "<tr><td><strong>Pulse rate:</strong></td>"
-	if(scan["pulse"] == -1)
+	if((scan["pulse"] == -1))
```
Comment (MACIEKBAKI) [inline code/_helpers/medical_scans.dm]: double (), why?

File: code/modules/SCP/SCPs/SCP-3349.dm [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+				heart.scp3349_induced = TRUE
+				to_chat(src, SPAN_INFO("You feel a sudden euphoric sensation!"))
+
+// SETUP
+
+GLOBAL_VAR(scp3349_precedentA)
+GLOBAL_VAR(scp3349_precedentB)
+
+GLOBAL_VAR(scp3349_fake_precedentA)
+GLOBAL_VAR(scp3349_fake_precedentB)
```
Comment (MACIEKBAKI) [inline code/modules/SCP/SCPs/SCP-3349.dm:33]: global variables my beloved

File: code/_helpers/medical_scans.dm (hunk 1 above)
Comment (aidan-bowman) [reply code/_helpers/medical_scans.dm]: remnant from prior code change, fixing

**Comment by aidan-bowman** [comment]
DON'T MERGE THIS YET I need to make some changes (I don't see a way to draft open PRs)

**Comment by TheDarkElites** [comment]
you should be able to, look harder

**Comment by aidan-bowman** [comment]
Alright, should be good for review

**Pull request merged**